test/integration/blocks-schema.test.js

```diff
@@ -3,22 +3,17 @@
  */
 import Ajv from 'ajv-draft-04';
 import glob from 'fast-glob';
-import path from 'path';
 
 /**
  * Internal dependencies
  */
 import blockSchema from '../../schemas/json/block.json';
 
 describe( 'block.json schema', () => {
-	const blockFolders = glob.sync( 'packages/block-library/src/*', {
-		onlyDirectories: true,
-		ignore: [ 'packages/block-library/src/utils' ],
-	} );
-	const testData = blockFolders.map( ( blockFolder ) => [
-		'core/' + path.basename( blockFolder ),
-		path.join( blockFolder, 'block.json' ),
-	] );
+	const jsonFiles = glob.sync(
+		[ 'packages/*/src/**/block.json', '{lib,phpunit,test}/**/block.json' ],
+		{ onlyFiles: true }
+	);
```

Comment:
Just making sure I'm following along: we're expanding the search to `block.json` files anywhere within packages or tests, so that we pick up the `block.json` files in the `widgets` and `edit-widgets` packages, and anywhere else we might add them in the future, so we don't need to remember to explicitly add those directories? Sounds good to me 👍